lib/catalog/order_item_sanitized_parameters.rb

```diff
@@ -76,7 +77,14 @@ def service_plan_does_not_exist?
     end
 
     def substitute(template)
-      Mustache.render(template, workspace)
+      template.gsub!(ORDER_ITEM_REGEX) do |order_item_name|
+        order_item_name.each_byte.map { |b| b.to_s(16) }.join
```

Comment:
I like what you did with the other snippet and turned it into the `encode_name` method, because at first glance, this looks very cryptic and doesn't make sense. Think we can do that here too? I would also prefer if we used `byte` instead of `b` just for searchability later down the road, but in the context of the line, `b` makes sense, so I'll leave it up to you.